test/javascript/tests/reader_acl.js

```diff
@@ -215,11 +215,6 @@ couchTests.reader_acl = function(debug) {
   );
         
   usersDb.deleteDb();
-  // have to delete the backside version now too :(
```

Comment:
Why are we removing these cleanup bits in tests?

Reply:
We used `15986` here. As you suggested, I changed to `_node` instead.